tests/stdlib/tnetpsk.nim

```diff
@@ -1,45 +1,35 @@
 discard """
   joinable:false
   batchable:false
-  action: "run"
   matrix: "--threads:on -d:ssl"
   targets: "c cpp"
   timeout:5
-  output:'''
-accepted connection
-connected with aethelfridda
-goodbye from bartholomew
-hello from aethelfridda
-identity hint "bartholomew"
-'''
-  sortoutput:true
 """
 import net
 from openssl import SSL_CTX_ctrl
 #using channels_builtin
-var serverChannel:Channel[bool]
+var serverChannel:Channel[Port]
 
 proc clientFunc(identityHint: string): tuple[identity: string, psk: string] =
-  echo "identity hint \"", identityHint,"\""
+  doAssert identityHint == "bartholomew"
   return ("aethelfridda","aethelfridda-loves-"&identityHint)
 
-proc client(){.thread.}=
+proc client(p:Port){.thread.}=
```

Comment:
```suggestion
proc client(p: Port){.thread.}=
```
(ditto elsewhere, also in things like:
```
#foo
=>
# foo
```
)

Reply:
i turned off nimpretty because it kept breaking random stuff in the stdlib and now all my code is hideous.

Reply:
we need `nimpretty --difflines` (refs https://github.com/nim-lang/Nim/issues/8458#issuecomment-457055195, like `git clang-format`), but at least you can try running nimpretty for just this test file?